packages/office-ui-fabric-react/src/components/SwatchColorPicker/SwatchColorPicker.tsx

```diff
@@ -214,4 +214,17 @@ export class SwatchColorPicker extends BaseComponent<ISwatchColorPickerProps, IS
       });
     }
   }
+
+  /**
+   * Get the white color cell style
+   * @param inputColor - The color of the current cell
+   * @returns - The White color cell style if the current style is white, empty otherwise
+   */
+  private _getWhiteCellStyle(inputColor: string): string {
```

Comment:
change this to string|undefined and don't typecast from the callers. This PR should not introduce any "as string" calls

Reply:
Okay, I got rid of the casting and instead used a non-null assertion operator. Hopefully this will be better.